sound/soc/sof/intel/hda-sdw-bpt.c

```diff
@@ -0,0 +1,352 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2024 Intel Corporation. All rights reserved.
+//
+
+/*
+ * Hardware interface for SoundWire BPT support with HDA DMA
+ */
+
+#include <sound/hdaudio_ext.h>
+#include <sound/hda-mlink.h>
+#include <sound/hda-sdw-bpt.h>
+#include <sound/sof.h>
+#include "../ops.h"
+#include "../sof-priv.h"
+#include "hda.h"
+
+#define BPT_FREQUENCY		192000
```

Comment:
need explanation of this value, I think it's 9.6MHz * Double data rate factor.
but if you can vary the clock now you'd need to use the max clock before using BRA/BPT...

Reply:
@plbossart I think it is the sampling rate of the HDA stream. But I am not sure how does it related to the SoundWire bus rate. The SDW bus clock and the HDA sampling rate seems to be independent to me.
I guess why `192000` is because that is the max value of [rate_bits[]](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/hda/hdac_device.c#L706).

Reply:
I can't recall where this came from... certainly something related to HDaudio format but why did we take 192k as the reference?